.github/actions/node-setup/action.yaml

```diff
@@ -40,3 +40,10 @@ runs:
         key: ${{ runner.os }}-turbo-${{ github.sha }}
         restore-keys: |
           ${{ runner.os }}-turbo-
+
+    - name: Export Package Manager
+      shell: bash
+      env:
+        package_manager: ${{ steps.detect-pm.outputs.package-manager }}
+      run: |
+        echo "NODE_PACKAGE_MANAGER=$package_manager" >> "$GITHUB_ENV"
```

Comment:
question: why do we need to export and ENV? can't we just use the action output in the consumer? ie.

```yaml
steps:
  - name: Setup Node.js
    id: node-setup
    uses: pagopa/dx/.github/actions/node-setup@main

  - name: FooBar
    .... ${{ steps.node-setup.outputs.package-manager }}
  ```